src/components/ImageView/index.native.js

```diff
@@ -25,59 +25,30 @@ const propTypes = {
     /** Function for handle on press */
     onPress: PropTypes.func,
 
-    ...windowDimensionsPropTypes,
+    /** Additional styles to add to the component */
+    style: PropTypes.oneOfType([PropTypes.arrayOf(PropTypes.object), PropTypes.object]),
 };
 
 const defaultProps = {
     isAuthTokenRequired: false,
     onPress: () => {},
+    style: {},
 };
 
-class ImageView extends PureComponent {
-    constructor(props) {
-        super(props);
-
-        this.state = {
-            isLoading: true,
-            imageWidth: 0,
-            imageHeight: 0,
-            interactionPromise: undefined,
-        };
-
-        // Use the default double click interval from the ImageZoom library
-        // https://github.com/ascoders/react-native-image-zoom/blob/master/src/image-zoom/image-zoom.type.ts#L79
-        this.doubleClickInterval = 175;
-        this.imageZoomScale = 1;
-        this.lastClickTime = 0;
-        this.amountOfTouches = 0;
-
-        // PanResponder used to capture how many touches are active on the attachment image
-        this.panResponder = PanResponder.create({
-            onStartShouldSetPanResponder: this.updatePanResponderTouches.bind(this),
-        });
-
-        this.configureImageZoom = this.configureImageZoom.bind(this);
-        this.imageLoadingStart = this.imageLoadingStart.bind(this);
-    }
-
-    componentDidUpdate(prevProps) {
-        if (this.props.url === prevProps.url) {
-            return;
-        }
+const DOUBLE_CLICK_INTERVAL = 175;
```

Comment:
I think we should keep this comment.
```suggestion
// Use the default double click interval from the ImageZoom library
// https://github.com/ascoders/react-native-image-zoom/blob/master/src/image-zoom/image-zoom.type.ts#L79
const DOUBLE_CLICK_INTERVAL = 175;
```

Reply:
Can we change it to 
```
const DOUBLE_CLICK_INTERVAL = ImageZoomProps.doubleClickInterval;
```
So we use always that value if they change it?